index.html

```diff
@@ -20,14 +20,36 @@
           width="20px"
         />
         <div id="myDropdown">
-          <a href="">Portoflio</a>
-          <a href="">About</a>
-          <a href="">Contact</a>
+          <a href="#works_section">Portoflio</a>
+          <a href="#about_me">About</a>
+          <a href="#contact">Contact</a>
           <a href=""
             ><img src="./images/Icon - Mail.png" alt="email-link" width="15"
           /></a>
         </div>
       </nav>
+      <div id="mobile-menu-modal" class="toogleModalDisplay">
```

Comment:
- [ ] to match the Figma design, your hamburger button should be aligned to the right, while you already used the `space-between` i suggest you remove the `margin: 0 4rem;` from #header_menu and it will be fixed 💯 

![image](https://user-images.githubusercontent.com/89282221/215511162-e7c74ef5-eb5c-4843-b39b-6ceac50d212f.png)

Reply:
This issue is well resolved! ✔️